Something/Database.EntityFramework/ApiContextFactory.cs

```diff
@@ -0,0 +1,22 @@
+﻿using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Design;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Database.EntityFramework
+{
+    public class ApiContextFactory : IDesignTimeDbContextFactory<ApplicationDbContext>
+    {
+        public ApiContextFactory()
+        {
+        }
+
+        public ApplicationDbContext CreateDbContext(string[] args)
+        {
+            var builder = new DbContextOptionsBuilder<ApplicationDbContext>();
+            builder.UseMySql("Server=127.0.0.1;Database=db;Uid=root;Pwd=mysql;");
```

Comment:
Not able to find a way to move to config. This is only used by EF for migration.

Reply:
I see, if i could move it I'll do